### PR TITLE
added contact and blog apps

### PR DESCRIPTION
**Pull request by mciray**
### Contact  Endpoint informations

127 0 0 1_8000_contact-api_contact (4)

### About  Endpoint informations

127 0 0 1_8000_about-api_biography (2)

### CODE REVIEW

File: backend/apps/about/serializers.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from .models import *
+
+
+class BiographySerializers(serializers.ModelSerializer):
+    class Meta:
+        model = Biography
+        fields = "__all__"
```
Comment (CoderMungan) [inline backend/apps/about/serializers.py:8]: gj

File: backend/apps/about/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import BiographyListView
+
+urlpatterns = [
+    path("biography", BiographyListView.as_view(), name="biography-list"),
+    
+]
```
Comment (CoderMungan) [inline backend/apps/about/urls.py:7]: gj

File: backend/apps/about/views.py [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
-
 # Create your views here.
+from rest_framework import generics
+from .serializers import BiographySerializers
+from .models import Biography
+
+
+class BiographyListView(generics.ListAPIView):
+    queryset = Biography.objects.all()  
+    serializer_class = BiographySerializers
```
Comment (CoderMungan) [inline backend/apps/about/views.py:10]: looking good

File: backend/apps/contact/models.py [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+from phonenumber_field.modelfields import PhoneNumberField
 # Create your models here.
 
 class Contact(models.Model):
     name = models.CharField(max_length=100)
     email = models.EmailField()
-    phone = models.CharField(max_length=20, blank=True)
-
+    phone = PhoneNumberField(blank=True,region='TR')
+
```
Comment (CoderMungan) [inline backend/apps/contact/models.py:9]: good detail about phones

File: backend/apps/contact/models.py [hunk 5]
```diff
@@ -1,12 +1,12 @@
 from django.db import models
-
+from phonenumber_field.modelfields import PhoneNumberField
```
Comment (CoderMungan) [inline backend/apps/contact/models.py:2]: Did u add requirements.txt too ?

File: backend/apps/contact/serializer.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import *
+
+class ContactSerializers(serializers.ModelSerializer):
+    class Meta:
+        model = Contact
+        fields = "__all__"
```
Comment (CoderMungan) [inline backend/apps/contact/serializer.py:7]: gj

File: backend/apps/contact/urls.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import ContactListView
+
+urlpatterns = [
+    path("contact", ContactListView.as_view(), name="contact-list"),
+]
```
Comment (CoderMungan) [inline backend/apps/contact/urls.py:6]: gj

File: backend/apps/contact/views.py [hunk 8]
```diff
@@ -1,3 +1,10 @@
...
-
 # Create your views here.
+from rest_framework import generics
+from .serializer import ContactSerializers
+from .models import Contact
+
+
+class ContactListView(generics.ListAPIView):
+    queryset = Contact.objects.all()  
+    serializer_class = ContactSerializers
```
Comment (CoderMungan) [inline backend/apps/contact/views.py]: Contact can be take `Post` method too can be revise please.

File: backend/backend/urls.py [hunk 9]
```diff
@@ -43,6 +43,8 @@
     path("admin/", admin.site.urls),
     path("api-auth/", include("rest_framework.urls")),
     path("blog-api/", include("apps.blog.urls")),
+    path("about-api/", include("apps.about.urls")),
+    path("contact-api/", include("apps.contact.urls")),
```
Comment (CoderMungan) [inline backend/backend/urls.py:47]: gj

**Review by CoderMungan — CHANGES_REQUESTED** [review]

File: backend/apps/contact/models.py (hunk 5 above)
Comment (mciray) [reply backend/apps/contact/models.py:2]: i forget to add that just need to add 
phonenumberslite==8.13.28

Comment (CoderMungan) [reply backend/apps/contact/models.py:2]: Can better good job!

File: backend/apps/contact/views.py (hunk 8 above)
Comment (mciray) [reply backend/apps/contact/views.py]: [c0a98ff](https://github.com/RythmSoft-Co/congenial-dollop/pull/6/commits/c0a98ff56662524fbbb5416417cbed423bd91970)

i Did it

Comment (CoderMungan) [reply backend/apps/contact/views.py]: gj

**Review by CoderMungan — APPROVED** [review]
LGTM!

**Pull request merged**